Makefile

```diff
@@ -8,7 +8,7 @@ endif
 TARGETS=awesome
 BUILDDIR=build
 
-all: $(TARGETS) ;
+all: check-unit $(TARGETS) ;
```

Comment:
I already :+1:'d this, so let's just merge this. ;-)

I do not see another way to do this.